docs/develop/office-versions-and-requirement-sets.md

```diff
@@ -32,7 +31,7 @@ In addition, other functionality such as add-in commands (ribbon extensibility)
 
 An add-in can only use APIs in requirement sets that are supported by the version of Office host where the add-in is running. To know exactly which requirement sets are available for a specific Office host version, refer to the following host-specific requirement set articles:
 
-- [Excel JavaScript API requirement sets](/office/dev/add-ins/reference/requirement-sets/excel-api-requirement-sets) (ExcelApi)
+- [Excel JavaScript API requirement sets](/office/dev/add-ins/reference/requirement-sets/excel-api-requirement-sets) (ExcelApi, [CustomFunctionsRuntime](../excel/custom-functions-overview.md))

```

Comment:
The link to custom functions overview is odd. This article has specific info about the runtime https://docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-architecture?view=office-js Link to this instead?